<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>0</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>Book title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Author</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Release year</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Group</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Available</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Taken by</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="grpSearch">
        <property name="title">
         <string>Search</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="lblBookname">
           <property name="text">
            <string>Book title: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lneSrch">
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QRadioButton" name="rdioAll">
               <property name="text">
                <string>All</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioNone">
               <property name="text">
                <string>None</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioBio">
               <property name="text">
                <string>Biography</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioBusiness">
               <property name="text">
                <string>Business</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioChildren">
               <property name="text">
                <string>Children</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioDic">
               <property name="text">
                <string>Dictionary</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioEngLit">
               <property name="text">
                <string>English literature</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioFiction">
               <property name="text">
                <string>Fiction</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="rdioHistory">
               <property name="text">
                <string>History</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioMed">
               <property name="text">
                <string>Medicine</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioMusic">
               <property name="text">
                <string>Music</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioNovel">
               <property name="text">
                <string>Novel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioPerLit">
               <property name="text">
                <string>Persian literature</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioReligious">
               <property name="text">
                <string>Religious</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioSciFic">
               <property name="text">
                <string>Sci-fic</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdioScience">
               <property name="text">
                <string>Science</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnSrch">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnAddBook">
          <property name="text">
           <string>Add book</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLendBook">
          <property name="text">
           <string>Lend book</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDeleteBook">
          <property name="text">
           <string>Delete book</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btrEdit">
          <property name="text">
           <string>Edit book</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>105</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnReturn">
          <property name="text">
           <string>Returned</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionAdd_book"/>
    <addaction name="actionRemove_book"/>
   </widget>
   <addaction name="menuManage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_book">
   <property name="text">
    <string>Add book</string>
   </property>
  </action>
  <action name="actionRemove_book">
   <property name="text">
    <string>Remove book</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
